README.md

```diff
@@ -1,251 +1,220 @@
-# stargate-mongoose
+# stargate-mongoose ![ci-tests](https://github.com/stargate/stargate-mongoose/actions/workflows/ci-tests.yml/badge.svg)
 
-![ci-tests](https://github.com/stargate/stargate-mongoose/actions/workflows/ci-tests.yml/badge.svg)
-
-`stargate-mongoose` is a mongoose driver for [JSON API Server](https://github.com/stargate/jsonapi)/[Astra DB](https://astra.datastax.com).
-
-## Table of contents
+`stargate-mongoose` is a mongoose driver for [JSON API Server](https://github.com/stargate/jsonapi) which runs on top of Apache Cassandra / DataStax Enterprise.
 
 1. [Quickstart](#quickstart)
-2. [Testing](#testing)
-3. [Compatability](#compatability)
+2. [Architecture](#architecture)
+3. [Features](#compatability)
```

Comment:
spelling: compatibility

Also, shouldn't the anchor should be renamed features anyway?

Or, I guess now we have Versions Compatibility and Features. Both should be in the ToC, right?

Reply:
Fixed.